explainaboard/metrics/accuracy.py

```diff
@@ -46,6 +66,86 @@ def calc_stats_from_data(
             )
         )
 
+    def _calc_metric_from_aggregate(
+        self,
+        agg_stats: np.ndarray[tuple[int], Any] | np.ndarray[tuple[int, int], Any],
+    ) -> np.ndarray[tuple[()], Any] | np.ndarray[tuple[int], Any]:
+        """Inner function of calc_metric_from_aggregate."""
+        if agg_stats.shape[-1] == 1:
+            return agg_stats.squeeze(-1)
+        return agg_stats
+
+    def calc_metric_from_aggregate(
+        self,
+        agg_stats: np.ndarray[tuple[int], Any] | np.ndarray[tuple[int, int], Any],
+    ) -> np.ndarray[tuple[()], Any] | np.ndarray[tuple[int], Any]:
+        """From aggregated sufficient statistics, calculate the metric value.
+
+        Args:
+            agg_stats: aggregated statistics. Shape must be:
+                - Non-batched data: [num_aggregate_stats]
+                - Batched data: [num_batches, num_aggregate_stats]
+
+        Returns:
+            Calculated metrics. Shape must be:
+                - Non-batched data: [] if num_aggregate_stats is 1.
+                    Otherwise [num_aggregate_stats].
+                - Batched data: [num_batches]
+        """
+        if agg_stats.ndim not in (self.stats_ndim(), self.stats_ndim() + 1):
+            raise ValueError(f"Invalid shape size: {agg_stats.shape}")
+
+        result = self._calc_metric_from_aggregate(agg_stats)
+        result_shape = (
+            () if agg_stats.shape[0] == self.stats_ndim() else (agg_stats.shape[0],)
+        )
+
+        assert result.shape == result_shape, (
+            "BUG: invalid operation: "
+            f"{type(self).__name__}._calc_metric_from_aggregate(): "
+            f"Expected shape {result_shape}, but got {result.shape}."
+        )
+
+        return result
+
+    def evaluate_from_stats(
```

Comment:
Don't override this function. Instead, directly implement statements to retrieve confidence scores in the original function (as just same as "score_ci" MetricValue).